src/Go/ui/GoPoint.tsx

```diff
@@ -15,11 +16,29 @@ interface GoPointProps {
   traditional: boolean;
   hover: boolean;
   valid: boolean;
-  emptyPointOwner: GoColor;
+  emptyPointOwner?: GoColor;
```

Comment:
This doesn't seem like it should be optional either. If needed, it can be `| undefined`.